hrui/CRM/HRUI/Upgrader/Steps/4701.php

```diff
@@ -20,7 +20,7 @@ public function upgrade_4701() {
   }
 
   private function up4701_setCustomCSSURL() {
-    $bootstrapcivicrmDirectory = CRM_Core_Resources::singleton()->getPath('org.civicrm.bootstrapcivicrm', 'css/custom-civicrm.css');
+    $bootstrapcivicrmDirectory = CRM_Core_Resources::singleton()->getPath('org.civicrm.shoreditch', 'css/custom-civicrm.css');
```

Comment:
well, this is not bootstrap anymore, so the variable name must be changed as well. I would suggest a more generic name, to avoid having to change it again in the future. Something like `$customCSSPath`